docs/develop/08_opendid/04_integrate_opendid.md

```diff
@@ -0,0 +1,117 @@
+-   `scope`: set value to `openid`.
+-   `state`: set to a secure random number.
+-   `nonce`: optional value, set to a secure random number.
+
+**Example**:
+
+```
+GET /api/v1/authorize?
+    response_type=code&
+    client_id=example-client&
+    redirect_uri=http://localhost:1606/callback.html&
+    scope=openid&
+    state=rkw49cbvd4azu5dsln1xbl&
+    nonce=vedur4om49ei8w91jt7wt HTTP/1.1
+```
+
+After successful authentication, the OpenDID service redirects back to the provided `redirect_uri` with `code` and `state` query parameters.
+
+**Example**:
+
+```
+/callback.html?
+    code=lwDS1ZpQBwR4Vdm53_L8bWpUJ1mx9A0mA_-86dubTqzqzwGazx1RyLX4Z_qf&
+    state=rkw49cbvd4azu5dsln1xbl
+```
+
+You can retrieve the `id_token` by calling the **POST** `/api/v1/token` and providing the following values in the form serialization:
+
+-   `code`: code value returned from `authorize`.
+-   `grant_type`: set value to `authorization_code`.
+-   `redirect_uri`: the same `redirect_uri` used in `authorize`.
+-   `client_id`: the client ID set in the `config.yaml` file.
+-   `client_secret`: the client secret value set in the `config.yaml` file.
+
+**Example**:
+
+```
+POST /api/v1/token HTTP/1.1
+Content-Type: application/x-www-form-urlencoded
+
+code=lwDS1ZpQBwR4Vdm53_L8bWpUJ1mx9A0mA_-86dubTqzqzwGazx1RyLX4Z_qf&
+grant_type=authorization_code&
+redirect_uri=http%3A%2F%2Flocalhost%3A1606%2Fcallback.html&
+client_id=example-client&
+client_secret=insecure_client_secret
+```
+
+The OpenDID service returns the `id_token` in the response body serialized as a JSON object.
+
+```json
+{
+    "access_token": "SsFhhSBMWsLeDMxVUVGreKARNwYxMZtGFfBr0-ZiH6iondSmwPRvQDqkG6Fh",
+    "token_type": "bearer",
+    "refresh_token": "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJkaWQ6a2lsdDo0b0VkNENVV3RwbkxUVnZENVBFd2lMUmlqMWdzQmprS1JMbVpES2lCOEdqN2I2V0wiLCJ3M24iOiJjdXN0b20iLCJleHAiOjE3MTY4MTYwNjQsImlhdCI6MTcxNjgxNTQ2NCwiaXNzIjoiZGlkOmtpbHQ6NHJzQkE3dEQ1S1E4TDlXSGpGallRdUhrTWtha2NmSGRDNUNhUVVjVXh5VWpEVkhBIiwiYXVkIjoiYXV0aGVudGljYXRpb24iLCJwcm8iOnsiRW1haWwiOiJhYmR1bEBraWx0LmlvIn0sIm5vbmNlIjoidmVkdXI0b200OWVpOHc5MWp0N3d0In0.yOmE_9jWKcAu8LpjVx7IsFyOOvlKbgo2oC4Imf-qrLY",
+    "id_token": "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJkaWQ6a2lsdDo0b0VkNENVV3RwbkxUVnZENVBFd2lMUmlqMWdzQmprS1JMbVpES2lCOEdqN2I2V0wiLCJ3M24iOiJjdXN0b20iLCJleHAiOjE3MTY4MTU1MjQsImlhdCI6MTcxNjgxNTQ2NCwiaXNzIjoiZGlkOmtpbHQ6NHJzQkE3dEQ1S1E4TDlXSGpGallRdUhrTWtha2NmSGRDNUNhUVVjVXh5VWpEVkhBIiwiYXVkIjoiYXBwbGljYXRpb24iLCJwcm8iOnsiRW1haWwiOiJhYmR1bEBraWx0LmlvIn0sIm5vbmNlIjoidmVkdXI0b200OWVpOHc5MWp0N3d0In0.YlRE9EGnSExQCb5m2iy4__58PZJlZdCZMsSvsuW4oj8"
+}
+```
+
+:::note
+In full-stack applications, calling the `token` endpoint is usually done through the back end to improve security.
+:::
+
+The `id_token` is a JSON web token (JWT) signed by the JWT key-pair specified in the `config.yaml` file of the OpenDID service. 
+You can verify this using the JWT public key, for example, by the back end of the Web app.
```

Comment:
I mean this should rather be a MUST than a CAN.

Reply:
```suggestion
You must verify this using the JWT public key, for example, by the back end of the Web app.
```